test/e2e/custom_url_test.go

```diff
@@ -466,18 +552,10 @@ func getComponentRouteSpec(t *testing.T, client *framework.ClientSet, routeName
 	return componentRouteSpec
 }
 
-func createTLSSecret(t *testing.T, client *framework.ClientSet, tlsSecretName, hostname string) {
+func createTLSSecret(t *testing.T, client *framework.ClientSet, tlsSecretName string) {
 	if tlsSecretName == "" {
 		return
 	}
-	tlsCert, err := crypto.MakeSelfSignedCAConfig(hostname, 1)
```

Comment:
have you checked whether it's possible in Go to create a certificate w/o SAN?

Reply:
so I was checking it but most of the time openssl was used to generate the certificate.